alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/TriggerService.kt

```diff
@@ -57,11 +61,48 @@ class TriggerService(val client: Client, val scriptService: ScriptService) {
     }
 
     // TODO: This is a placeholder to write MonitorRunner logic, it can be replaced with the actual implementation when available
+    @Suppress("UNCHECKED_CAST")
     fun runAggregationTrigger(
         monitor: Monitor,
         trigger: AggregationTrigger,
         ctx: AggregationTriggerExecutionContext
     ): AggregationTriggerRunResult {
-        return AggregationTriggerRunResult(triggerName = trigger.name, error = null, aggregationResultBuckets = mapOf())
+        return try {
+            val bucketIndices =
+                ((ctx.results[0]["aggregations"] as HashMap<*, *>)[trigger.id] as HashMap<*, *>)["bucket_indices"] as List<*>
```

Comment:
Define or use existing string constants for `"aggregation"` and `"bucket_indices"`

Reply:
Updated to use constants wherever possible in TriggerService